How to buy software in an enterprise - mega guide (#9171)
PostHog/posthog.com: contents/founders/how-to-buy-enterprise-software.md

---
date: 2024-08-15

> title: How to buy enterprise software

Author: ```suggestion
title: How to buy software as an enterprise
```

- Negotiating internally with your team is _much_ more important and difficult than negotiating with the salesperson. 
- The typical enterprise software buying process involves 8-12 people - it’s not just you and a finance person who signs it off.
- Your salesperson can do _so_ much more for you than just show you the product and negotiate pricing.

Are you a software buying newbie at a large company? Have you bought software before and it went terribly and took way longer than you expected? Have you been handed something called an RFP and have no idea WTF to do with it? Do you enjoy reading enterprise software buying guides in your spare time?

This guide is for you! We'll break down the entire process - everything you need to do to look good internally, but also how to handle your salesperson. As a bonus, we've also thrown in the 7 final bosses you'll (probably) need to beat at the end. 

## The 5 levels you need to clear

Buying enterprise software is a game, and games have levels (wait, haven't we done this [before](/founders/product-market-fit-game)?)

1. I think I need software to solve a problem
2. Shortlist the ones that could do the job

> 3. Get demos & test the ones you like

Author: ```suggestion
3. Get demos and test the ones I like
```

4. It’s time to negotiate
5. How do I actually buy this thing?

You need to complete all five levels to win. At each stage, we'll outline what to do internally, what to do with your salesperson, and a common trap to avoid. 

Let's go!

> ### Level 1 - I think I need software to solve a problem

Reviewer: ```suggestion
### Level 1: I think I need software to solve a problem
```

Reviewer: amend here and throughout. i'd always opt for : over – as it's neater and more standard. you've used – on all the crossheads, i'd switch 'em for :

Send over the following info by email in advance of any call to make the best use of your time too - not only will you spend more time actually in the product, but the demo you get will be more relevant:

- What’s the problem you have, and what you’re doing today to address it
- Must have requirements - short as possible
- When you want a solution in place
- Who you are and if you have the ability to purchase yourself - be honest here, they will still want to talk to you even if you don’t have authority to buy software, as you will be their deal ‘champion’!

**It's a trap!** 

Review websites are a waste of time. Any company that has a good presence on them has either paid for them or gamed the system by only asking happy customers for reviews. However, reading bad reviews can be insightful.

<NewsletterForm />

### Level 3 - Get demos & test the ones you like

![pmf-guide](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/blog/pmf-game-guide/pmf-level-3.png)

**Internal process**

> It's time to loop in the right people for the demos. Your manager might be interested, but unless it’s really business critical there's no point bringing someone very senior to this demo.

Author: ```suggestion
It's time to bring in the right people for the demos. Your manager might be interested, but unless it’s really business critical there's no point bringing someone very senior to this demo.
```

- What they care about - liability, ability to terminate the contract, general legal point scoring
- How to win them over - thank them profusely for taking a ‘commercial’ angle and not being overly legalistic

**The CEO who thinks we should go with something else because they met the founder at Davos**
- When to engage - you won’t get to engage them probably, but be prepared to justify your choice at no notice
- What they care about - their LinkedIn following
- How to win them over - presentation deck, 10 slides, few words, legible on mobile

**The CFO who thinks we should use spreadsheets for everything**
- When to engage - right at the beginning, bizarrely (if you can even reach them)
- What they care about - the share price
- How to win them over - make the economic case, put numbers against time saved, revenue generated etc. 

**The generic loud person on a totally different team who will only sign in once but give lots of opinions**
- When to engage - whenever they spring up, you’ll need to be reactive
- What they care about - having their voice heard
- How to win them over - get on a call, reassure them their feedback will be taken seriously, then throw away your notes

> **The digital transformation guru whose responsibilities are incredibly vague but their solution is always AI**
> - When to engage - whenever really, they are good for helping you build hype internally
> - What they care about - getting the company to do new stuff
> - How to win them over - sympathize with them about how resistant to change the organization is

Reviewer: ```suggestion
**The digital transformation guru whose responsibilities are incredibly vague but their solution is always AI**
- When to engage: whenever really, they are good for helping you build hype internally
- What they care about: getting the company to do new stuff
- How to win them over: sympathize with them about how resistant to change the organization is
```

> ## Sure there's a better way?

Reviewer: is this wording right? i don't quite get it... are we saying

Are you sure there's not a better way?

or something else?

Author: ohhh, should be 'surely' I will amend

> ## Sure there's a better way?

Author: ```suggestion
## Surely there's a better way?
```

---

> date: 2024-08-15

Author: ```suggestion
date: 2024-08-14
```

## The 8 final bosses you’ll (probably) need to beat

As you go through this process, you will probably encounter one or more of these people. Here are some quick tips on how to deal with them. [LOTTIE ARTWORK TO BE ADDED]

**The procurement person who actually has a personality if you get to know them**
- When to engage - when you have selected your vendor of choice.
- What they care about - following the proper internal process,  and getting the software for as cheap as possible.
- How to win them over - frame your request as asking for their help/expertise, not telling them what to do. Come prepared with all the notes on the evaluation so far.

**The engineer who implemented the current solution and is taking it personally that you’re suggesting something else**
- When to engage - right at the beginning

> - What they care about - saving face if you replace what they implemented

Reviewer 2: ```suggestion
- What they care about - saving face if you replace what they implemented , and not losing any important functionality by changing software.
```
